### PR TITLE
Build: Name "HAVE"-style constants consistently

### PR DESCRIPTION
**Pull request by nrwahl2**
Closes T542

### CODE REVIEW

**Review by clumens — COMMENTED** [review]
I haven't tested "Closes" before, but I know that "Fixes:" in the commit message will do what you expect in phabricator.  Beyond that, this all looks good to me.

**Comment by kgaillot** [comment]
> I haven't tested "Closes" before, but I know that "Fixes:" in the commit message will do what you expect in phabricator. Beyond that, this all looks good to me.

Yep, "closes" works. The syntax is quite flexible, see https://secure.phabricator.com/T5132

File: configure.ac [hunk 1]
```diff
@@ -780,7 +780,7 @@ AS_CASE(["$host_os"],
         ])
 
 AC_SUBST(INIT_EXT)
-AC_DEFINE_UNQUOTED([SUPPORT_PROCFS], [$PROCFS],
+AC_DEFINE_UNQUOTED([HAVE_PROCFS], [$PROCFS],
```
Comment (kgaillot) [inline configure.ac:783]: FYI this is fine, but the reason I suggested HAVE_LINUX_PROCFS is that some other OSes actually do have a procfs (it originated in Solaris I believe) but they do not follow the same path naming conventions as Linux, which are the only ones Pacemaker supports

**Review by kgaillot — APPROVED** [review]
Thanks

File: configure.ac (hunk 1 above)
Comment (nrwahl2) [reply configure.ac:783]: Not intentional, just on autopilot :)

https://github.com/ClusterLabs/pacemaker/pull/2831

**Comment by nrwahl2** [comment]
> Yep, "closes" works. The syntax is quite flexible, see https://secure.phabricator.com/T5132

Interesting that "spites" is an accepted keyword for closing a task.

**Pull request merged**